frontend/benefit/applicant/src/components/applications/forms/application/step3/attachmentsList/AttachmentsList.tsx

```diff
@@ -0,0 +1,61 @@
+import { $PrimaryButton } from 'benefit/applicant/components/applications/Applications.sc';
+import { ATTACHMENT_TYPES } from 'benefit/applicant/constants';
+import { IconPlus } from 'hds-react';
+import camelCase from 'lodash/camelCase';
+import noop from 'lodash/noop';
+import * as React from 'react';
+
+import AttachmentItem from './attachmentItem/AttachmentItem';
+import { $Container, $Heading, $Message } from './AttachmentsList.sc';
+import { useAttachmentsList } from './useAttachmentsList';
+
+export interface AttachmentsListProps {
+  attachmentType: ATTACHMENT_TYPES;
+  showMessage?: boolean;
+}
+
+const AttachmentsList: React.FC<AttachmentsListProps> = ({
+  attachmentType,
+  showMessage,
+}) => {
+  const { t, translationsBase, attachments } = useAttachmentsList();
+  return (
+    <$Container>
+      <$Heading>
+        {t(`${translationsBase}.types.${camelCase(attachmentType)}.title`)}
+      </$Heading>
+      {attachments.length > 0 ? (
+        <>
```

Comment:
This fragment tag is not redundant here.

Reply:
yes, i ll remove